create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+
+    From your DigitalOcean account go to the [API](https://cloud.digitalocean.com/account/api) section in order to generate the tokens. 
+    
+    Create a personal access token first by clicking the `Generate New Token` button:
+
+    ![Applications & API Key](content/img/pt_gen_1st.png)
+
+    then make sure to give it a proper name and that the `Write` scope is checked:
+
+    ![Applications & API Key](content/img/pt_gen_2nd.png)
+
+    In the end we should have something like this:
+
+    ![Applications & API Key](content/img/api_access_key.png)
+
+    Copy the token value and save it in a local environment variable as we will need it later on (make sure to replace the `<>` placeholder):
+
+    ```bash
+    export TF_VAR_do_token="<your_do_personal_access_token>"
+    ```
+    Next we navigate to `Spaces access keys` and click the `Generate New Key` button:
+
+    ![Spaces Key](content/img/spk_gen_1st.png)
+
+    then give it a proper name and mark it as done:
+
+    ![Spaces Key](content/img/spk_gen_2nd.png)
+
+    In the end we should have something like this:
+
+    ![Spaces Key](content/img/spaces_key.png)
+
+    Copy the key and secret value and save each in a local environment variable for later use (make sure to replace the `<>` placeholder):
+
+    ```bash
+    export DO_SPACES_ACCESS_KEY="<your_do_spaces_access_key>"
+    export DO_SPACES_SECRET_KEY="<your_do_spaces_secret_key>"
+    ```
+3. Going further we have to create a [DO Space](https://cloud.digitalocean.com/spaces) for storing the Terraform state file. Go ahead to your DigitalOcean account panel and click on the `Create` button (the green one) from the upper right corner. Select a region closer to you then make sure that `Restrict file listing` is checked and finally give it a proper name. The required steps are highlighted below:
+
+    ![Create DO Space](content/img/do_spaces_rs.png)
+4. Clone this repository on your local machine and navigate to the appropriate directory:
+   
+    ```bash
+    git clone https://github.com/digitalocean/container-blueprints.git
+    cd create-doks-with-terraform-flux
+    ```
+5. Terraform initialization must be perfomed next. A [DO Spaces](https://cloud.digitalocean.com/spaces) bucket for storing the Terraform state file is highly recommended because we do not have to worrry about exposing sensitive data as long as the space is private of course. Another advantage is that the state of our infrastructure is backed up so we can re-use it in order to do a refresh and change only the affected parts which is a great and powerful feature of Terraform in the end. Having a common shared space across more team members is desired as well in order to perform collaborative work via Terraform.
```

Comment:
```suggestion
5. Initialize Terraform. A [DO Spaces](https://cloud.digitalocean.com/spaces) bucket for storing the Terraform state file is highly recommended because we do not have to worry about exposing sensitive data as long as the space is private of course. Another advantage is that the state of our infrastructure is backed up so we can re-use it in order to do a refresh and change only the affected parts which is a great and powerful feature of Terraform in the end. Having a common shared space across more team members is desired as well in order to perform collaborative work via Terraform.
```